src/auth-service/models/Client.js

```diff
@@ -227,4 +228,14 @@ ClientSchema.methods = {
   },
 };
 
-module.exports = ClientSchema;
+const ClientModel = (tenant) => {
```

Comment:
@BenjaminSsempala , now all models export the model instead of the Schema.

Reply:
Thanks @Baalmart